jupyter-docker/Dockerfile

```diff
@@ -52,8 +48,6 @@ RUN echo "deb $DEBIAN_REPO/debian jessie main\ndeb $DEBIAN_REPO/debian-security
     ca-certificates \
     wget \
     curl \
-    maven \
-    git \
     build-essential \
     lsb-release \
 
```

Comment:
I think all the Python things (i.e. the apts and the pips) should be together.

Reply:
👍